server.js

```diff
@@ -2,6 +2,7 @@ const express = require('express')
 const app = express()
 const morgan = require('morgan')
 const bodyParser = require('body-parser')
+const dotenv = require('dotenv').config()
```

Comment:
What's the point of holding the result of `config` in a variable?

Reply:
Will change this